app/controllers/stories_controller.rb

```diff
@@ -334,7 +337,7 @@ def stories_by_timeframe
     elsif params[:timeframe] == "latest"
       @stories.where("score > ?", -20).order(published_at: :desc)
     else
-      @stories.order(hotness_score: :desc).where("score > 2")
+      @stories.order(hotness_score: :desc).where("score > 2") # Andy: need to discuss this
```

Comment:
So here is another place where we have a score of `2` that we check for. I'm not sure if this is a place that we need to make configurable too, but wanted to highlight it in case we do.